lib/duo.c

```diff
@@ -324,6 +327,49 @@ _duo_bson_response(struct duo_ctx *ctx, bson *resp)
     return (ret);
 }
 
+static int
+_sleep_ms(long milliseconds)
+{
+    struct timespec tim;
+    tim.tv_sec = milliseconds / 1000;
+    tim.tv_nsec = (milliseconds % 1000) * 1000000;
+
+    if (nanosleep(&tim, &tim) != 0) {
```

Comment:
@vbscott can you help me remember what standard functions we have available in Duo Unix.
nanosleep appears to be in `POSIX.1-2001` is that sufficient?

Reply:
I believe so; sleep(), which is used later in this file, is also `POSIX.1-2001`.